cmd/geth/config.go

```diff
@@ -47,10 +47,10 @@ var (
 	dumpConfigCommand = &cli.Command{
 		Action:      dumpConfig,
 		Name:        "dumpconfig",
-		Usage:       "Show configuration values",
-		ArgsUsage:   "",
+		Usage:       "Export configuration values in a TOML format",
+		ArgsUsage:   "[dumpfile]",
```

Comment:
```suggestion
		ArgsUsage:   "<dumpfile (optional)>",
```

Reply:
No need to use <angle brackets(optional)> since square brackets indicate this arg is optional.
If we use angle brackets for optional args like this, why we need square brackets?

Reply:
> since square brackets indicate this arg is optional.

Oh, that is something I did not know. I don't think we've used that way of writing ArgsUsage in other places in go-etherum

Reply:
I think a lot of users, like me, are not aware of the difference between `[foo]` and `<foo>`, so for clarity's sake let's spell it out: `<dumpfile (optonal)>`

Reply:
Could you review #26729 for this?
Github closed this PR automatically when I sync `go-ethereum` to the HEAD...